### ISSUE TITLE
Beginner's documentation needed

### ISSUE THREAD

**Issue by bestouff** [Pocco81/DAPInstall.nvim#73]
Hi,

dap-buddy looks cool, but how does a total noob uses this ? Let me say I have a fully configured NVIM (via LunarVim) and I want to debug some C code; I added dap+dap-buddy via `lvim.builtin.dap.active = true` but now ... how do I debug (e.g.) C code ?

**Comment by Mate2xo** [comment]
I guess we're not supposed to use it anymore, as it's being rewritten (?) and the docs seem to have vanished with the rebranding of this repo. 
Try https://github.com/williamboman/mason.nvim , or configure it yourself with https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation

**Comment by bestouff** [comment]
Yeah, problem is I'm using LunarVim and `dap-buddy` + `nvim-lsp-installer` is what's used by default here...
